test/parallel/test-assert.js

```diff
@@ -1,13 +1,11 @@
 'use strict';
 require('../common');
-var assert = require('assert');
-var a = require('assert');
+const assert = require('assert');
+const a = require('assert');
 
 function makeBlock(f) {
-  var args = Array.prototype.slice.call(arguments, 1);
-  return function() {
-    return f.apply(this, args);
-  };
+  const args = [].slice.call(arguments, 1);
+  return () => f.apply(this, args);
```

Comment:
This change actually make `this` refer to something else, so I wouldn’t convert it to an arrow function.